sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/AnalysisHelper.scala

```diff
@@ -250,6 +250,29 @@ trait AnalysisHelper extends QueryPlan[LogicalPlan] { self: LogicalPlan =>
     }
   }
 
+  /**
+   * Recursively transforms the expressions of a tree, first to all of its
```

Comment:
I think we should highlight that the traversal order is bottom-up for both logical plan treea and expression tree.

Reply:
Tried to rephrase the wording to emphasize the `bottom-up` over all the expressions of all the tree node.